src/redux/categories/categorySlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  category: [],
+  design: 'Under Construction!',
+};
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+  },
+});
+
+export default categorySlice.reducer;
```

Comment:
- [ ] Nice work so far 👍🏿 but as per project requirement this is how you can correct this slice

 - Add Missing Initial State Field:
In the initial state of your code, add a field called design with the value 'Under Construction!'.

- Adjust Reducer Logic:
Modify the reducer logic inside your createSlice to update the design field based on the action payload. Here's the adjusted logic:

```
checkStatus: (state, action) => {
  state.design = action.payload === "Under construction" ? "Under Construction!" : state.design;
},
```
- Import and Export Correctly:
Make sure you are importing and exporting the createSlice function correctly from @reduxjs/toolkit.

- Export the Reducer:
At the end of your module, export the reducer using the .reducer property of the createSlice result, just like you did in the original code.